pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/ElasticTestUtil.java

```diff
@@ -301,4 +323,21 @@ public static void removeDsmField(String esIndex, String ddpParticipantId, Strin
         UpsertPainless upsert = new UpsertPainless(esIndex, QueryBuilders.termsQuery("_id", ddpParticipantId));
         upsert.export(script, Map.of(), field);
     }
+
+    public static void addParticipantDsmFromFile(String esIndex, String fileName, String ddpParticipantId) {
+        Gson gson = new Gson();
+        try {
+            String dsmJson = TestUtil.readFile(fileName);
+            Dsm dsm = gson.fromJson(dsmJson, Dsm.class);
+            ObjectMapper mapper = new ObjectMapper();
+            Map<String, Object> valueMap = mapper.convertValue(dsm, Map.class);
+            Map<String, Object> dsmMap = Map.of("dsm", valueMap);
+            ElasticSearchUtil.updateRequest(ddpParticipantId, esIndex, dsmMap);
+        } catch (Exception e) {
+            e.printStackTrace();
+            throw new RuntimeException(e);
+//            Assert.fail("Unexpected exception creating profile for participant " + ddpParticipantId);
+        }
```

Comment:
Creates a participant with the specified data in the profile, so we can use filters to get exactly this participant in the tests.